packages/frontend/src/components/markdown-split-editor/markdown-split-editor.tsx

```diff
@@ -166,7 +169,19 @@ export const MarkdownSplitEditor = ({
   return (
     <Flex {...flexProps} direction="column">
       <Flex direction="row" justify="flex-end" pr="0.25rem">
-        {showFormattingHelp && <FormattingHelp />}
+        <Stack direction="row" spacing="1rem" align="stretch">
```

Comment:
Swap for an `<HStack>`, it's the same thing but has the row direction baked in.

```suggestion
        <HStack spacing="1rem" align="stretch">
```